src/webgpu/api/validation/buffer/create.spec.ts

```diff
@@ -26,21 +28,56 @@ g.test('size')
         kBufferSizeAlignment * 2,
       ])
   )
-  .unimplemented();
+  .fn(t => {
+    const { mappedAtCreation, size } = t.params;
+    const isValid = !mappedAtCreation || size % kBufferSizeAlignment === 0;
+    const usage = BufferUsage.COPY_SRC;
+    t.expectGPUError(
+      'validation',
+      () => t.device.createBuffer({ size, usage, mappedAtCreation }),
+      !isValid
+    );
+  });
+
+const BufferUsage = GPUConst.BufferUsage;
+const BufferUsageWithEmptyAndInvalid = { ...BufferUsage, NO_USAGE: 0, INVAILD: 0x8000 } as const;
+const listBufferUsage = Object.keys(BufferUsage) as [keyof typeof BufferUsage];
```

Comment:
This is similar to `kBufferUsages` from `capability_info.ts` except it uses the key names instead of the numeric values. I think using the numeric values is OK here.

Reply:
Done.